### ISSUE TITLE
Why does Potlatch deny a changeset regarding node 4396753945?

### ISSUE THREAD

**Issue by tracker51** [systemed/potlatch3#51]
Hello,

based on my own view on site and two NTRIP corrected tracks I tried to do some slight corrections of positions using Potlatch, but without success: Potlatch denied to save the changeset. Reason:  "Error creating changeset" In the end I have been successful using ID editor. 

Since I can't rule out that it's a mistake on my part, I've documented the attempt to make a tiny correction of a tag belonging to this node:
[https://www.openstreetmap.org/node/4396753945](url)
Indeed, the bus stop's name is "Masurenweg" (Masurenway) instead of "Masurenstraße" (Masurenstreet). But  even this slight correction of a kind of typo isn't accepted by Potlatch:

![image](https://github.com/user-attachments/assets/1ac61333-476b-4e52-b4fc-b23d30bd53d6)

Has anyone here an explanation? Many thanks in advance!

Best regards

tracker51

**Comment by systemed** [comment]
Are you using the latest version that has been updated for the OAuth changes on openstreetmap.org?

**Comment by tracker51** [comment]
Indeed, I didn't know anything about a new Potlatch version. In the meantime I have found the place where to download it. But, unfortunately, I'm not a software specialist! Therefore I'm not able to decide which file I should download: with or without CPU rendering. I would really appreciate it if someone made a recommendation. Many thanks in advance!

Regards

tracker51

**Comment by tracker51** [comment]
So, I just tried it out. I uninstalled Potlatch 3, downloaded AIR runtime - version 51.0.1.3 and Cross-platform AIR file and installed both. After starting Potlatch 3.1, I mapped a planned addition (hunting stand) and tried to save it, but unfortunately to no avail: the required login to OSM was not possible.

Then I remembered a note when downloading the AIR runtime:

![Potlatch3 1_benötigte_AIR-Version](https://github.com/user-attachments/assets/34b09ffc-84b8-4ad0-be34-eb48860a54e9)

I then uninstalled AIR runtime - version 51.0.1.3 and Cross-platform AIR file, downloaded AIR runtime including WebKit - version 33.1.1.744, installed this version and the Cross-platform AIR file again.

After starting Potlatch 3.1, I mapped the planned addition (hunting stand) again and tried to save it: this time it worked immediately, no prior login was necessary! Hurray!

But unfortunately there is now a new issue. When I tried to save further additions or changes a few minutes later, the following message appeared:

![Potlatch3 1_Changeset-Nachfrage](https://github.com/user-attachments/assets/7ece1c81-3950-46cd-86cd-29679d25fdfc)

After confirming with "Yes" I had to justify my changes again. The same thing happened again later after further additions/changes. Or should I better have clicked "No"?

Am I doing something wrong or is Potlatch 3.1 so forgetful that - unlike before - it always asks for detailed justifications for subsequent changes/additions. It's kind of annoying!

Many thanks for any help in advance!

Best regards

tracker51
